### PR TITLE
[main] Add Konflux configurations

### PR DESCRIPTION
**Pull request by serverless-qe**
Add Konflux components and pipelines

### CODE REVIEW

**Comment by pierDipi** [comment]
/hold

Until we cut 1.34

File: .konflux/applications/serverless-operator-release-134/components/imagerepositories/openshift-knative-serverless-operator-serverless-bundle-release-134.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ImageRepository
+metadata:
+  annotations:
+    image-controller.appstudio.redhat.com/update-component-image: "true"
+  labels:
+    appstudio.redhat.com/application: serverless-operator-release-134
+    appstudio.redhat.com/component: serverless-bundle-main
```
Comment (pierDipi) [inline .konflux/applications/serverless-operator-release-134/components/imagerepositories/openshift-knative-serverless-operator-serverless-bundle-release-134.yaml:8]: This should be `serverless-bundle-release-134`

**Review by creydr — APPROVED** [review]
/lgtm

**Comment by creydr** [comment]
/unhold
as we need to create the components first (before we add the pipelines (https://github.com/openshift-knative/serverless-operator/pull/2824))

**Pull request merged**